sis_import_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh --
```

Comment:
You and Pushyami should have the build scripts on the same level.  IMHO the second level directory with the files for just one project would be a good place.